protocol/errorsHelp.json

```diff
@@ -1,28 +1,32 @@
 {
-  "40000": "https://support.ably.io/solution/articles/3000082819-error-code-40000-bad-request",
```

Comment:
@tomczoink you shouldn't have mixed a commit that changed the URLs for existing error codes, with a commit that added new error codes.  It's harder to review, and harder someone down the line to see what this commit was actually about.